### PR TITLE
Fix tests that were not changed when underlying code changed

### PR DESCRIPTION
**Pull request by jamesr66a**
This should resolve some spurious build breakage i've been seeing in my PRs

### CODE REVIEW

File: test/expect/TestOperators.test_addmm.expect [hunk 1]
```diff
@@ -4,10 +4,20 @@ producer_version: "0.2"
 domain: "com.facebook"
 graph {
   node {
+    input: "3"
+    output: "5"
+    op_type: "expand"
```
Comment (houseroad) [inline test/expect/TestOperators.test_addmm.expect:9]: This does not look good.

File: test/expect/TestOperators.test_concat2.expect [hunk 2]
```diff
@@ -6,10 +6,10 @@ graph {
   node {
     input: "1"
     input: "2"
-    output: "3"
-    op_type: "Concat"
+    output: "4"
+    op_type: "cat"
```
Comment (houseroad) [inline test/expect/TestOperators.test_concat2.expect:10]: Also why now exports "cat", not "Concat"?

**Review by houseroad — COMMENTED** [review]
It seems the pytorch exporter is broken or the pytorch version is not right.

File: test/expect/TestOperators.test_addmm.expect (hunk 1 above)
Comment (ezyang) [reply test/expect/TestOperators.test_addmm.expect:9]: Yeah, the `expand` is the broadcasting stuff which I am working on fixing.

File: test/expect/TestOperators.test_concat2.expect (hunk 2 above)
Comment (ezyang) [reply test/expect/TestOperators.test_concat2.expect:10]: Yeah, looks like we have lost the symbolic for cat somehow.

**Pull request closed without merging**